main/main.cpp

```diff
@@ -1922,3 +1922,7 @@ void Main::cleanup() {
 	OS::get_singleton()->clear_last_error();
 	OS::get_singleton()->finalize_core();
 }
+
+bool Main::is_project_manager() {
+	return project_manager;
+}
```

Comment:
No new line here. Not sure what our code style rules say about this, but clang-format doesn't complain.

Reply:
Our rules say UNIX-style is king ;)